reduction.py

```diff
@@ -3,13 +3,37 @@
 import pandas as pd
 import numpy as np
 import sklearn
-
-# summary_image of image function that tells for a specific function what the evaluation provides
-
+# summary_image of image function that tells for a specific function what the evaluation provides 
 # summary_all_images()
 
 # count_user_evaluations
 
+def import_img_data(fn,img_num):
+    '''Read CSV file and create DataFrame for specific image
+
+    Parameters
+    ----------
+    fn : string
+         CSV file name
+    img_num: int
+             Image number (at most the number of unique images in file)
+
+    Returns
+    -------
+    img_id: int
+            Image ID
+    img_df: DataFrame
+            DataFrame for image
+    '''
+
+    df = pd.read_csv(fn)
+    subject_id = df['subject_id']
+    ids = subject_id.unique()
+    ids = sorted(ids)
+    img_id = ids[img_num]
+    img_df = df.loc[subject_id == img_id]
+    
+    return img_id, img_df
```

Comment:
It could be good to add the number of evaluations of that tasks as an output to this function. 
Or alternative provide another function that counts the number of outputs.